stable/hlf-peer/templates/deployment.yaml

```diff
@@ -201,9 +203,11 @@ spec:
             - mountPath: /var/hyperledger/tls/ord/cert
               name: ord-tls-rootcert
             {{- end }}
-            {{- if .Values.secrets.channel }}
-            - mountPath: /hl_config/channel
-              name: channel
+            {{- if .Values.secrets.channels }}
+            {{- range .Values.secrets.channels}}
```

Comment:
@inzamam-iqbal Please add extra space here, for readability, and indent the range block.